### PR TITLE
Fixing documentation for messaging and activity feed notifications

### LINKED ISSUE #11498: To use Application permission team needs to be in migrate state
After quite some searching online why the Application permission didn't work I can across this:
https://stackoverflow.com/questions/65234475/getting-unauthorized-on-create-chatmessage-even-though-application-has-teamwork

When you want to use the Application token permission you need to have the team in the migrate state, this isn't mentioned anywhere in this doc and can cause quite some head scratching to why it doesn't work.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 01f9643c-c7a2-0187-b4c5-48c67acd444d
* Version Independent ID: 25c34dc2-250e-4fa6-67a6-dfe396109526
* Content: [Create chatMessage in a channel - Microsoft Graph beta](https://docs.microsoft.com/en-us/graph/api/channel-post-message?view=graph-rest-beta&amp%3Btabs=http&tabs=http)
* Content Source: [api-reference/beta/api/channel-post-message.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/beta/api/channel-post-message.md)
* Product: **microsoft-teams**
* Technology: **microsoft-graph**
* GitHub Login: @RamjotSingh
* Microsoft Alias: **MSGraphDocsVteam**

### PR DESCRIPTION
**Pull request by RamjotSingh**
Following changes have been made

- Making it clear that Teamwork.Migrate.All is only supported for migration purposes (messaging and teams)
- Adding note specifying that only standard template is supported for migration team
- Adding sample payloads to messaging change notifications
- Adding FAQs to activity feed notifications doc

Fixes #11498

### CODE REVIEW

File: concepts/teams-changenotifications-chatmessage.md [hunk 1]
```diff
@@ -252,6 +252,107 @@ Content-Type: application/json
 }
 ```
 
+## What should you expect in the notification
```
Comment (AkJo) [inline concepts/teams-changenotifications-chatmessage.md:255]: This should be a sub-heading of example 4 I would think.

Comment (AkJo) [reply concepts/teams-changenotifications-chatmessage.md:255]: Or nvm. I see that this page is dedicated to notifications.

File: concepts/teams-send-activityfeednotifications.md [hunk 2]
```diff
@@ -258,3 +258,27 @@ Microsoft Teams users can customize the notifications they see in their feed, as
...
+
+## FAQs
+
+##### Who needs the Teams app to be installed?
+
+Target user must have the Teams app which is sending notifications installed.
+
+##### Can a user send notifications to themselves?
+
+No, at the moment a user cannot send notifications to themselves. For scenario like these, please use application permissions.
```
Comment (AkJo) [inline concepts/teams-send-activityfeednotifications.md:270]: For scenarios like these

File: concepts/teams-send-activityfeednotifications.md [hunk 3]
```diff
@@ -258,3 +258,27 @@ Microsoft Teams users can customize the notifications they see in their feed, as
...
+
+##### Can a Teams app control how the notifications are shown to user (banner, feed etc)
+
+No, only users are allowed to change notification settings.
+
+##### I installed my app, but I do not see notification settings under the user account
+
+The settings will appear once the first notification has been sent by the Teams app. This reduces the amount of settings users see and help in better management.
```
Comment (AkJo) [inline concepts/teams-send-activityfeednotifications.md:278]: that the users see

File: concepts/teams-send-activityfeednotifications.md [hunk 4]
```diff
@@ -258,3 +258,27 @@ Microsoft Teams users can customize the notifications they see in their feed, as
...
+
+##### I started getting a Conflict error, how do I resolve it
```
Comment (AkJo) [inline concepts/teams-send-activityfeednotifications.md:280]: either lowercase Conflict or put it in italics or `Conflict`

**Review by AkJo — APPROVED** [review]

File: concepts/teams-send-activityfeednotifications.md (hunk 4 above)
Comment (RamjotSingh) [reply concepts/teams-send-activityfeednotifications.md]: This is a title, so I put 409 and then conflict in bracket.

File: concepts/teams-send-activityfeednotifications.md (hunk 3 above)
Comment (RamjotSingh) [reply concepts/teams-send-activityfeednotifications.md]: Done.

File: concepts/teams-send-activityfeednotifications.md [hunk 5]
```diff
@@ -258,3 +258,27 @@ Microsoft Teams users can customize the notifications they see in their feed, as
...
+
+The biggest reason for conflict errors is because multiple Teams apps installed in the scope (team, chat, user etc) have the same AzureAD appId being used in `webApplicationInfo` section of the manifest. When this happens you will get an error like `Found multiple applications with the same AAD App ID 'Your AzureAD AppId'.`. In such cases, ensure that you use unique AzureAD apps for unique Teams apps. However, having the same Teams app installed in multiple scopes (team + user for example) is not an issue.
```
[the remark this answers is not in the document]
Comment (RamjotSingh) [reply concepts/teams-send-activityfeednotifications.md]: Rephrased a bit.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**